stages/org.osbuild.ovf

```diff
@@ -31,23 +34,23 @@ OVF_TEMPLATE = """<?xml version="1.0"?>
         <vssd:ElementName>Virtual Hardware Family</vssd:ElementName>
         <vssd:InstanceID>0</vssd:InstanceID>
         <vssd:VirtualSystemIdentifier>image</vssd:VirtualSystemIdentifier>
-        <vssd:VirtualSystemType>vmx-15</vssd:VirtualSystemType>
+        <vssd:VirtualSystemType>{vsphere_virtual_system_type}</vssd:VirtualSystemType>
       </System>
       <Item>
         <rasd:AllocationUnits>hertz * 10^6</rasd:AllocationUnits>
         <rasd:Description>Number of Virtual CPUs</rasd:Description>
         <rasd:ElementName>2 virtual CPU(s)</rasd:ElementName>
         <rasd:InstanceID>1</rasd:InstanceID>
         <rasd:ResourceType>3</rasd:ResourceType>
-	<rasd:VirtualQuantity>2</rasd:VirtualQuantity>
+        <rasd:VirtualQuantity>{ovf_cpu_count}</rasd:VirtualQuantity>
```

Comment:
I'm ok with making CPU/memory into a variable, albeit an optional one (in the manifest schema). As Im pretty sure users can change this when importing the ovf.

Reply:
It turns out that even though this was templatized in our template we pretty much always passed in 2 cpu and 4096 memory anyway so I might just not templatize it for now.